pages/samvera/developer_resources/collections/hyrax_2.1/collection-types.md

```diff
@@ -0,0 +1,168 @@
+
+**Admin Sets** (preconfigured)
+
+An administrative grouping of works that an administrative unit is ultimately responsible for managing. The set itself helps to manage the items within it. (See [Admin Sets as Collections FAQ](admin-sets-as-collections-faq.html) for more information.)
+
+**User Collections** (preconfigured)
+
+An intellectual grouping of works, primarily used by individual users to create groups of items or favorites. 
+
+**Exhibits**
+
+A grouping of existing works into an exhibit oriented toward display of content to end users.
+
+**General Nested**
+
+Provide the ability to nest collections for the purpose of organizing content.  Several use cases fall into this category and have significant overlap of requirements.  These include DSpace migrated content (i.e. Community → Collection), organization unit based collections (e.g. University → School → Department; Agency → Sub-agency; etc.), and general nesting for organizing materials.  
+
+ 
+Don’t see your use case?  That’s ok, because Collection Types allow you to define and configure your own type.
+
+### Configuring a Collection Type
+
+You can add as many collection types as you need for your system.<sup>1</sup>  Before we start configuring types, let's look at the configuration options that define the behaviors of a collection type.
+
+**Basic configurations:**
+
+* NESTABLE Allow collections to be nested (a collection can contain other collections)
+* MULTIPLE MEMBERSHIP Allow a work to belong to multiple collections
+* DISCOVERY Allow collections to be discoverable
+* SHARING Allow users to assign collection managers, depositors, and viewers for collections they manage
+* REQUIRE MEMBERSHIP<sup>2</sup> A work must belong to at least one collection of this type
+
+**Advanced configurations:<sup>3</sup>**
+
+* WORKFLOW Allow collections of this type to assign workflow to a new work
+* VISIBILITY  Allow collections of this type to assign initial visibility settings to a new work
+
+**Proposed but not implemented**
+
+* CONTROL VISIBILITY  Visibility of works in collections of this type are controlled by the settings in the collection
+
+
+**Collection Participants:**
+
+* assign Collection Managers (groups or users) who can edit collections other users have created, including adding to and removing works from a collection, modifying collection metadata, and deleting collections
+* assign Collection Creators (groups or users) who can create and manage their own collections
+
+
+<font color="gray">Footnotes:</font>
+
+<sup>1</sup>  Based on use cases identified so far, it is expected that the number of collection types will be less than 5.  There isn't a limit on the number of collection types you can create, but you might want to consider this in your application design.
```

Comment:
This footnote feels too vague to be useful. You might want to consider what in your application design? Why?